doc/src/guides/custom-functions.md

```diff
@@ -1437,7 +1437,7 @@ existing SymPy logic defined on `1 - cos(x)`).
 ...             return True
 ...
 ...     def _eval_is_positive(self):
-...         # versin(x) is positive iff x is real and not an even multiple of pi
+...         # versin(x) is positive if x is real and not an even multiple of pi
```

Comment:
Are you sure it shouldn't be iff here? (As in "if and only if") Then we may prefer to spell it out though...

Reply:
A counter example is
```python
In [18]: (1 - cos(pi+I))
Out[18]: 1 + cosh(1)
```

Reply:
Yeah, see https://github.com/sympy/sympy/pull/23488#discussion_r906656356